include/openssl/core_dispatch.h

```diff
@@ -66,8 +66,8 @@ OSSL_CORE_MAKE_FUNC(int,core_get_params,(const OSSL_CORE_HANDLE *prov,
 # define OSSL_FUNC_CORE_THREAD_START           3
 OSSL_CORE_MAKE_FUNC(int,core_thread_start,(const OSSL_CORE_HANDLE *prov,
                                            OSSL_thread_stop_handler_fn handfn))
-# define OSSL_FUNC_CORE_GET_LIBRARY_CONTEXT    4
-OSSL_CORE_MAKE_FUNC(OPENSSL_CORE_CTX *,core_get_library_context,
+# define OSSL_FUNC_CORE_GET_LIBCTX    4
```

Comment:
Indentation prior to "4" is wrong

Reply:
Addressed with c089c4c8ce3b139dd1a5e543f563e9ba8b280e4c